include/llvm/Support/ELF.h

```diff
@@ -756,8 +756,8 @@ enum : unsigned {
 
   SHF_MASKOS = 0x0ff00000,
 
-  SHF_IMM_RANDADDR = 0x00100000,
-  SHF_IMM_RANDWRAPPER = 0x00200000,
+  // Section location in memory should be randomized by the loader.
```

Comment:
I think we should use "Section base address can be randomized independently by the loader"